multi_import/importer.py

```diff
@@ -137,7 +145,9 @@ class Importer(object):
     export_filename = None
 
     cached_query = CachedQuery
-    serializer = None
+    serializer = None  # Deprecated in favour of serializer_class
```

Comment:
are we keeping this for backwards compatibility?

Reply:
Yeah, I kept it for back-compatibility - at least until @bellkelly's MR is merged.